wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -43,7 +44,6 @@ class SweepRun:
     queued_run: Optional[public.QueuedRun] = None
     args: Optional[Dict[str, Any]] = None
     logs: Optional[List[str]] = None
-    program: Optional[str] = None
```

Comment:
we aren't using the program for launch sweeps anymore, this feels as good a time as any to remove it.